src/Avalonia.Base/Utilities/MathUtilities.cs

```diff
@@ -267,6 +267,36 @@ public static int Clamp(int val, int min, int max)
             }
         }
 
+        /// <summary>
+        /// Converts an angle in degrees to radians.
+        /// </summary>
+        /// <param name="angle">The angle in degrees.</param>
+        /// <returns>The angle in radians.</returns>
+        public static double Deg2Rad(double angle)
```

Comment:
do these conversion methods get called so often might be worth inlining?

Reply:
It will inline automatically just fine.